### PR TITLE
1-8 requested improvements

### PR DESCRIPTION
**Pull request by lauras5**
issues #3 
- also changed some func names

### CODE REVIEW

File: client/src/TagComponent.svelte [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<script>
+    import { deleteTag } from 'sql-tag-system-fetch-utils';
```
Comment (ALMaclaine) [inline client/src/TagComponent.svelte:2]: This should be split into two components.

TagList/TagView
Tag

Don't put Component in the name

File: client/src/TagComponent.svelte [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    }
+
+</script>
+
+{#each tags as {id, tag}}
+    <div transition:fade class='inline-flex m-2 bg-gray-300 p-1 pl-4 pr-1 rounded-2xl'>
+        <span>{tag}</span>
+        <span class="ml-4 bg-gray-300 rounded-full w-6 h-6 flex justify-center items-center leading-6" on:click={handleDelete(id)}>
+              <Icon class="myClass1 myClass2" icon={faTimes}></Icon>
```
Comment (ALMaclaine) [inline client/src/TagComponent.svelte:24]: what are these classes

File: client/src/TagComponent.svelte [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            console.log(`Error: ${e}`)
+        }
+    }
+
+</script>
+
+{#each tags as {id, tag}}
+    <div transition:fade class='inline-flex m-2 bg-gray-300 p-1 pl-4 pr-1 rounded-2xl'>
+        <span>{tag}</span>
+        <span class="ml-4 bg-gray-300 rounded-full w-6 h-6 flex justify-center items-center leading-6" on:click={handleDelete(id)}>
```
Comment (ALMaclaine) [inline client/src/TagComponent.svelte:23]: Mmm i dont think that click handler should work, its run on creation you need a arrow function () => handleDelete

File: client/src/App.svelte [hunk 4]
```diff
@@ -1,67 +1,60 @@
...
-	import { fade } from 'svelte/transition';
-	import {addTag, getTag, listTags, deleteTag} from 'sql-tag-system-fetch-utils';
+	import { onMount } from 'svelte';
+	import { addTag, listTags } from 'sql-tag-system-fetch-utils';
+    import TagComponent from './TagComponent.svelte';
+
     let tags = [];
     let newTag = '';
-    let limit = 200;
+    let limit = 100;
```
Comment (ALMaclaine) [inline client/src/App.svelte:8]: Unless this is changed in the UI it shouldn't be a global variable, it's only used in the handleAddMoreFunction

File: client/src/App.svelte [hunk 5]
```diff
@@ -1,67 +1,60 @@
...
     let page = 0;
```
Comment (ALMaclaine) [inline client/src/App.svelte:9]: same with page

File: client/package.json [hunk 6]
```diff
@@ -6,6 +6,7 @@
     "concurrently": "^5.3.0",
     "cross-env": "^5.2.1",
     "css-loader": "^2.1.1",
+    "fa-svelte": "^3.1.0",
```
Comment (ALMaclaine) [inline client/package.json:9]: Is this for sure a dev dep? Is that in the docs? Have you tried building and seeing if it works outside of dev?

**Review by ALMaclaine — CHANGES_REQUESTED** [review]

File: client/src/TagComponent.svelte (hunk 2 above)
Comment (lauras5) [reply client/src/TagComponent.svelte:24]: - removed classes

File: client/src/TagComponent.svelte (hunk 3 above)
Comment (lauras5) [reply client/src/TagComponent.svelte:23]: updated to arrow function

File: client/package.json (hunk 6 above)
Comment (lauras5) [reply client/package.json:9]: - docs say to install as dev dependency, works outside on build too

File: client/src/App.svelte (hunk 5 above)
Comment (lauras5) [reply client/src/App.svelte:9]: moved to handleAddMore function, i was unsure if we needed to edit values in the future

**Review by ALMaclaine — APPROVED** [review]
lgtm

**Comment by ALMaclaine** [comment]
closes #3

**Comment by ALMaclaine** [comment]
closes #3

**Pull request merged**